ios/CoinbaseWalletSDK/Task/TaskManager.swift

```diff
@@ -35,16 +37,28 @@ class TaskManager {
         return true
     }
     
-    func findRequest(for response: EncryptedResponseMessage) -> RequestMessage? {
+    static func findRequest(for response: EncryptedResponseMessage) -> RequestMessage? {
```

Comment:
how about
```suggestion
    static func findRequest(for requestId: UUID) -> RequestMessage? {
```
i think this will be more flexible